docs/examples/interpolated-periods.py

```diff
@@ -0,0 +1,68 @@
+"""
+Interpolated Data Periods
+===================
+
+Identifying periods in a time series where the data has been
+linearly interpolated.
+"""
+
+# %%
+# Identifying periods where time series data has been linearly interpolated
+# and removing these periods may help to reduce noise when performing future
+# data analysis. This example shows how to use
+# :py:func:`pvanalytics.features.gaps.interpolation_diff`, which identifies and
```

Comment:
Here, elsewhere in this file, and in the stale data file, I think this should be `quality` instead of `features`

Reply:
Changed in 
commit 839c794a37982d4b12951cbb283d33774f27bb03